### ISSUE TITLE
Broke gulp-cssnano

### ISSUE THREAD

**Issue by joaomoreno** [ben-eb/postcss-normalize-url#11]
Hey I use `gulp-cssnano` in the project. My builds started failing just now. I see you published this module about 20 mins ago. Related?

```
2016-02-10T16:59:07.511Z: [command]/usr/local/bin/gulp mixin --gulpfile /vso/work/2/s/gulpfile.js
2016-02-10T16:59:11.030Z: module.js:338
2016-02-10T16:59:11.030Z:     throw err;
2016-02-10T16:59:11.030Z:     ^
2016-02-10T16:59:11.030Z: Error: Cannot find module 'postcss-normalize-url'
2016-02-10T16:59:11.030Z:     at Function.Module._resolveFilename (module.js:336:15)
2016-02-10T16:59:11.030Z:     at Function.Module._load (module.js:278:25)
2016-02-10T16:59:11.030Z:     at Module.require (module.js:365:17)
2016-02-10T16:59:11.030Z:     at require (module.js:384:17)
2016-02-10T16:59:11.030Z:     at Object.<anonymous> (/vso/work/2/s/node_modules/gulp-cssnano/node_modules/cssnano/index.js:29:26)
2016-02-10T16:59:11.030Z:     at Module._compile (module.js:460:26)
2016-02-10T16:59:11.030Z:     at Object.Module._extensions..js (module.js:478:10)
2016-02-10T16:59:11.030Z:     at Module.load (module.js:355:32)
2016-02-10T16:59:11.030Z:     at Function.Module._load (module.js:310:12)
2016-02-10T16:59:11.030Z:     at Module.require (module.js:365:17) 
2016-02-10T16:59:11.031Z:     at require (module.js:384:17)
```

**Comment by TrySound** [comment]
@joaomoreno Try to remove node_modules folder and install again please.

**Comment by cehoffman** [comment]
Related I think. `postcss-normalize-url` wasn't published with index.js. More specifically the package.json doesn't set the lib folder to dist which has index.js.

**Comment by pmowrer** [comment]
I'm having the exact same problem. Cleaning `node_modules` has no effect.

**Comment by TrySound** [comment]
Ops. I see. It's main field. @ben-eb  :)

**Comment by ben-eb** [comment]
Fixed in `3.0.7`. Sorry, automated tests don't catch these. :smile:

**Comment by joaomoreno** [comment]
:+1:
